intro/classes/index.html

```diff
@@ -11,8 +11,6 @@
     <link rel="stylesheet" href="../../styles/code_tags.css">
 </head>
 <body>
-    <h1>{{ page.title }}</h1>
-    <hr>
```

Comment:
Can you undo this edit? It should be fixed in the branch I'm writing this article in and I don't necessarily want to have to solve a merge conflict for a problem I've already fixed. (I guess my branch has been unmerged for a while, so I should probably open the PR for merging so that things can be reconciled where needed)

Reply:
Sure, I'll undo it when I get home

Reply:
Fixed